app-forensics/yara/files/yara-4.2-test.patch

```diff
@@ -0,0 +1,11 @@
+--- a/tests/test-rules.c
```

Comment:
Remember to include links to upstream bug reports + gentoo bugs + relevant upstream commits etc at top of patches